### PR TITLE
first version of s3Uploader

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,54 +1,80 @@
...
 
+group = 'com.github.bibsysdev'
+version = '0.1.1'
+
 
-project.ext{
-    nvaCommonsVersion = "1.5.2"
-    awsSdkVersion ="1.11.979"
+project.ext {
+    nvaCommonsVersion = "7899c8fb7af504e8688a1c68db9521f3713eb6fa"
```
Comment (axthosarouris) [inline build.gradle:30]: temporary version until code is merged in nva-commons

File: jitpack.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#This file runs only when one asks a version or Git hash-version from Jitpack
```
Comment (axthosarouris) [inline jitpack.yml:1]: copied from nva-datamodel-java

File: src/main/java/no/unit/cristin/RestAuthorizer.java [hunk 3]
```diff
@@ -1,33 +0,0 @@
-package no.unit.cristin;
```
Comment (axthosarouris) [inline src/main/java/no/unit/cristin/RestAuthorizer.java:1]: not used anymore

File: src/test/java/HttpTest.java [hunk 4]
```diff
@@ -1,51 +0,0 @@
-import static org.hamcrest.MatcherAssert.assertThat;
```
Comment (axthosarouris) [inline src/test/java/HttpTest.java:1]: not used

File: template.yml [hunk 5]
```diff
@@ -1,475 +0,0 @@
-AWSTemplateFormatVersion: '2010-09-09'
```
Comment (axthosarouris) [inline template.yml:1]: not used

**Review by brinxmat — APPROVED** [review]

**Pull request merged**